source/tools/entry.js

```diff
@@ -6,6 +6,16 @@ const EntryProperty = {
     Username:                     "username"
 };
 
+const EntryDefaultAttribute = {
+    PassModifiedTime:              "passModifiedTime",
+    PassExpiryTime:                "passExpiryTime",
+    PassExpiry:                    "passExpiry"
```

Comment:
I was thinking that we store in `passExpiry`, the number of days before expiration and 0 if there is no password expiration. This is set by the user, so it's in human readable format (in seconds, it will be more difficult for the user). 
I remove `passExpiryTime` and calculate when we need it (`hasExpiredPassword()`).
I keep `passExpiry` in days format, to keep it in human readable format. 
If you think it's better to have it in seconds or anything else, tell me, i will do the changes.